src/screens/CreateTokenScreen.tsx

```diff
@@ -102,7 +102,9 @@ export const CreateTokenScreen = ({ route, navigation }: Props) => {
     return null
   }
 
-  console.log(expoToken)
+  if (typeof process.env.JEST_WORKER_ID === 'undefined') {
```

Comment:
minor, but I would suggest that we use [this one instead](https://jestjs.io/docs/environment-variables#node_env) which feels a bit more generic than this jest-specific env var